.github/workflows/check-before-merging.yml

```diff
@@ -1,8 +1,9 @@
 name: Automated checks
```

Comment:
Is it worth including workflows for cypress, attaching PR's to Trello & dependabot PR merge?

Reply:
Definitely, thanks for suggesting. Will add these to the follow-up ticket